source/Plugins.cpp

```diff
@@ -96,6 +108,8 @@ const Plugin *Plugins::Load(const string &path)
 
 	plugin->path = path;
 
+	plugin->version = Files::Read(path + "version.txt");
```

Comment:
Should this really be a separate file instead of inside the plugin.txt file?

Reply:
changed it